src/gui/guiKeyChangeMenu.cpp

```diff
@@ -221,12 +223,7 @@ void GUIKeyChangeMenu::drawMenu()
 	video::IVideoDriver* driver = Environment->getVideoDriver();
 
 	video::SColor bgcolor(140, 0, 0, 0);
-
-	{
-		core::rect < s32 > rect(0, 0, 745, 620);
-		rect += AbsoluteRect.UpperLeftCorner;
-		driver->draw2DRectangle(bgcolor, rect, &AbsoluteClippingRect);
-	}
```

Comment:
Wondering why this was removed?

Reply:
Not needed, this makes this file consistent with the other `GUIModalMenu` derived menus. It also removes the hard-coded dimensions.

Reply:
Fine for me but will keep this line comment open for other s to see.